coalib/coala_modes.py

```diff
@@ -61,8 +59,10 @@ def mode_json(args, debug=False):
 
     if args.show_bears:
         try:
-            local_bears, global_bears = get_filtered_bears(
-                args.filter_by_language, LogPrinter())
+            from coalib.parsing.Filter import Filter
```

Comment:
Updated. Surprisingly its not able to resolve the reference of `Filter`. Logged at https://ci.appveyor.com/project/coala/coala/build/1.0.8117/job/78fcbqt9qp6atmyj#L507.